### PR TITLE
[Docs] Fix store file meta data docs.

### PR DESCRIPTION
**Pull request by howardhuanghua**
Fix store file meta data docs.

### CODE REVIEW

File: docs/changelog/91188.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+pr: 91188
```
Comment (kingherc) [inline docs/changelog/91188.yaml:1]: I believe we don't need a PR .yaml changelog for doc changes. Can you remove this file please?

**Review by kingherc — CHANGES_REQUESTED** [review]
Thanks for the PR. One minor comment.

File: docs/changelog/91188.yaml (hunk 1 above)
Comment (howardhuanghua) [reply docs/changelog/91188.yaml]: Removed. Thank you.

**Comment by kingherc** [comment]
@elasticmachine please test this

**Review by kingherc — APPROVED** [review]
LGTM, thanks!

**Comment by kingherc** [comment]
@elasticmachine run elasticsearch-ci/part-2

**Comment by kingherc** [comment]
@elasticmachine run elasticsearch-ci/part-3

**Comment by kingherc** [comment]
@elasticmachine update branch

**Comment by kingherc** [comment]
Hi @howardhuanghua , it seems your PR's branch is many commits behind the elastic:main branch. Can you update it? So that the tests run successfully before I merge it.

**Comment by howardhuanghua** [comment]
> Hi @howardhuanghua , it seems your PR's branch is many commits behind the elastic:main branch. Can you update it? So that the tests run successfully before I merge it.

Thank you @kingherc . I have merge the latest master.

**Comment by kingherc** [comment]
@elasticmachine test this please

**Comment by kingherc** [comment]
@elasticmachine run elasticsearch-ci/part-3

**Pull request merged**